source/fundamentals/connection/connection-options.txt

```diff
@@ -0,0 +1,414 @@
+.. _csharp-connection-options:
+
+==================
+Connection Options
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This section describes the MongoDB connection and authentication options
+available in the {+driver-short+}. You can configure your connection using either 
+the connection URI or a ``MongoClientSettings`` object.
+
+.. _csharp-connection-uri:
+
+----------------------------------
+Using the Connection URI
+----------------------------------
+
+If you pass a connection URI to the ``MongoClient`` constructor, you can include
+connection options in the string as ``<name>=<value>`` pairs. In the following example,
+the connection URI contains the ``connectTimeoutMS`` option with a value of ``60000``
+and the ``tls`` option with a value of ``true``: 
+
+.. literalinclude:: /includes/fundamentals/code-examples/connection/LocalConnectionConfig.cs
+   :language: csharp
+   :dedent:
+
+.. _csharp-mongo-client-settings:
+
+-----------------------------
+Using ``MongoClientSettings`` 
+-----------------------------
+
+You can use a ``MongoClientSettings`` object to configure connection settings in code
+rather than in a connection URI. Configuring the connection this way makes it easier to 
+change settings at runtime, helps you catch errors during compilation, and provides 
+more configuration options than the connection URI.
+
+To use a ``MongoClientSettings`` object, create an instance of the class, set 
+its properties, and pass it as an argument to the ``MongoClient`` constructor:
+
+.. literalinclude:: /includes/fundamentals/code-examples/connection/MongoClientSettingsConfig.cs
+   :language: csharp
+   :dedent:
+
+Connection Options
+------------------
+
+The following table lists each connection option available in the
+``MongoClientSettings`` class and, if possible, how to achieve the same result in 
+the connection string. If a ``MongoClientSettings`` property maps to more than one 
+option in the connection string, the **Connection URI Example** column shows all 
+relevant options. 
+
+.. note::
+
+  If you're using a query parameter for a time duration, the value must be in 
+  milliseconds. For example, to specify 60 seconds, use the value ``60000``. If you're 
+  using a ``MongoClientSettings`` object for a time duration, use the appropriate 
+  ``TimeSpan`` value.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 30 30
+
+   * - **MongoClientSettings** Property
```

Comment:
no logging configuration for mongoClientSettings?

Reply:
Good catch.

Reply:
Can you explain?

Reply:
Currently MongoClientSettings provides a way to configure logging. We probably need to describe it

Reply:
Can you point me to some information?

Reply:
cc: @BorisDog

Reply:
Yes, `LoggingSettings` property.
https://mongodb.github.io/mongo-csharp-driver/2.18/reference/driver/logging/

Reply:
Leaving to a [later ticket](https://jira.mongodb.org/browse/DOCSP-26773), as mentioned by Jordan [here.](https://mongodb.slack.com/archives/C03RFE0V9NV/p1673306249775959?thread_ts=1673305947.079499&cid=C03RFE0V9NV)

Reply:
I would still suggest to add the `LoggingSettings` property even with most basic description like "Client logging settings". Just have it visible for the users, as it's quite valuable monitoring tool.

Agreed that we can have the more detailed examples and info done in later ticket.